examples/bin/supervise

```diff
@@ -179,13 +195,28 @@ usage() unless GetOptions(
   'vardir|d=s',
   'kill-timeout|t=i',
   'chdir=s',
-  'svlogd:s'
+  'svlogd:s',
+  'array|a=s{,}'
```

Comment:
I have a question regarding your suggestion. How will this line `@config_lines = @{${'commands'}}` work given `commands` is not declared? 

Did you mean something like this?

```
usage() unless GetOptions(
  \%opt,
  'conf|c=s',
  'vardir|d=s',
  'kill-timeout|t=i',
  'chdir=s',
  'svlogd:s',
  'command=s@' => \@commands
);

usage() unless ((@commands || $opt{'conf'}) && $opt{'vardir'});

my @config_lines;

# get commands to execute either from reading the config file or command line
if (not defined $opt{'conf'}) {
    @config_lines = @commands
} else {
    @config_lines = read_config_file($opt{'conf'});
}
```

Reply:
I don't think there is a need to declare a target for each argument since `\%opt` is the first arg here. GetOptions has two modes:

1. Provide a hashref (like `\%opt`) first; all options get written as keys to that hashref.
2. Provide a target for each option.

We're using (1) here.

Reply:
It is working for me now, I was trying to access it like `$opt{'command'}` instead of `@{$opt{'command'}}`.